db/version_edit.cc

```diff
@@ -247,6 +254,7 @@ const char* VersionEdit::DecodeNewFile4From(Slice* input) {
   uint64_t file_size;
   SequenceNumber smallest_seqno;
   SequenceNumber largest_seqno;
+  bool has_global_seqno = false;
```

Comment:
If we want to have a general solution, we may want a more straight-forward name for what it is. Maybe something like 'ignore_seqno_in_file' or something like that?

Reply:
'largest_seqno_is_global_seqno' maybe better?

Reply:
I feel 'ignore_seqno_in_file' is better than "largest_seqno_is_global_seqno".